avocado_vt/plugins/vt_resolver.py

```diff
@@ -0,0 +1,61 @@
+import warnings
+
+from avocado.core.nrunner import Runnable
+from avocado.core.plugin_interfaces import Resolver
+from avocado.core.resolver import (ReferenceResolution,
+                                   ReferenceResolutionResult)
+from avocado.core.settings import settings
+
+from ..discovery import DiscoveryMixIn
+
+
+class VTResolver(Resolver, DiscoveryMixIn):
+
+    name = 'vt'
+    description = 'Test resolver for Avocado-VT tests'
+
+    def _parameters_to_runnable(self, params):
+        params = self.convert_parameters(params)
+        url = params.get('name')
```

Comment:
@richtja Do you mean `uri` here as in URI? This is more compatible with the arguments used by runnable and URL could be conceptually confusing.

Reply:
:facepalm: yes definitely, good catch